src/reqwesp/error.rs

```diff
@@ -0,0 +1,191 @@
+use std::error::Error as StdError;
```

Comment:
What's the point of this file?

Reply:
Creating and returning more useful and readable errors for certain functions.